lib/intercom-rails/auto_include_filter.rb

```diff
@@ -79,7 +78,7 @@ def intercom_script_tag
           nonce = CoreExtensions::IntercomRails::AutoInclude.csp_nonce_hook(controller)
           options.merge!(:nonce => nonce)
         end
-        @script_tag = ScriptTag.new(options)
+        @script_tag ||= ScriptTag.new(options)
```

Comment:
Ha... I caught the same thing #275, but this entire repo has been radio silence for months.

Reply:
Ah totally missed that! Yeah the whole thing seem a bit of a mess really. Surprising given that Intercom is such a great product.